packages/api/src/schema/api-schema.yaml

```diff
@@ -286,20 +286,36 @@ components:
       properties:
         playbackId:
           type: string
-          description: Playback ID of the stream or asset to clip
+          description: >-
+            The playback ID of the stream or stream recording to clip. Asset
+            playback IDs are not supported yet.
         startTime:
           type: number
-          description:
-            Start time of the clip in UNIX timestamp (in milliseconds)
+          description: >-
+            The start timestamp of the clip in Unix milliseconds. This is the
+            **server time** - not relative time to the start of the stream. _See
+            the ClipTrigger in the UI Kit for an example of how this is
+            calculated (for HLS, it uses `Program Date-Time` tags, and for
+            WebRTC, it uses the latency from server to client at stream
+            startup)._
         endTime:
           type: number
-          description: End time of the clip in UNIX timestamp (in milliseconds)
+          description: >-
+            The end timestamp of the clip in Unix milliseconds. This is the
+            **server time** - not relative time to the start of the stream. _See
+            the ClipTrigger in the UI Kit for an example of how this is
+            calculated (for HLS, it uses `Program Date-Time` tags, and for
+            WebRTC, it uses the latency from server to client at stream
+            startup)._
         name:
           type: string
-          description: Name of the clip
+          description: >-
+            The optional friendly name of the clip to create.
         sessionId:
           type: string
-          description: Session ID of the stream to clip
+          description: >-
+            The optional session ID of the stream to clip. This can be used to
```

Comment:
Maybe worth explaining the relationship between this and playbackId. I guess that if you dont provide this it clips the stream otherwise it clips a session? And it could alternatively be referenced by the session recording asset playbackId as well? (I don't know, these are actually questions about how it works too haha)

Reply:
Haha yeah let me get feedback from @emranemran in here as well.

Reply:
Yeah maybe we should clarify here that sessionId should be used to clip recordings otherwise it will clip the livestream?